ui/static/body-three-cols.scss

```diff
@@ -1,57 +0,0 @@
-@import "media_queries";
-@import "variables";
-
-.three-cols{
-  display: grid;
-  grid-template-areas: none;
-  grid-template-columns: 1fr;
-  font-size: #{$font-size-m};
-
-  @include desktop {
-    grid-template-areas: 'left middle rigth';
-    grid-template-columns: 20% 60% 20%;
-    font-size: #{$font-size-d};
-  }
-}
-
-.cat-grid{
-  display: grid;
-  grid-template-columns: repeat(auto-fill, 20%);
-  grid-column-gap: 1em;
-  grid-row-gap: 2em;
-  justify-items: stretch;
-}
-
-.cat-grid a{
-  text-decoration: none;
-  color: black;
-  border: 1px solid black;
-  border-radius: .3em;
-  width: 100%;
-  text-align: center;
-  padding: 2em 0 2em 0;
-  justify-self: center;
-}
-
-.auth-grid{
-
-  display: grid;
-  grid-column-gap: 1em;
-  grid-row-gap: 1em;
-  justify-items: center;
-  margin: 0 2em 0 2em;
-}
-
-.auth-grid a{
-  text-decoration: none;
-  color: black;
-  border-width: 0 0 0 10px;
-  border-style: solid;
-  border-radius: .3em;
-  width: 100%;
-  text-align: center;
-  padding: .5em 0 .5em 0;
-  justify-self: center;
-  background-color: #e5e5e5;
-  margin: 0 2em 0 2em;
-}
```

Comment:
este archivo se usa en el  
welldone-BE/articles/templates/articles/users.html

Reply:
lo he cambiado en users.html

Reply:
me di cuenta luego, por eso la resolvi